code/style.css

```diff
@@ -1,7 +1,247 @@
+  font-size: 60px;
+}
+
+.text p {
+  font-size: 16px;
+  margin-bottom: 20px; 
+  padding: 10px;
+}
+
+fieldset {
+  display: block;
+  margin-inline-start: 1px;
+  margin-inline-end: 1px;
+  margin-top: 5em;
+}
+
+.grid-container {
+   display: grid;
+   grid-template-columns: repeat (2, 1fr);
+   grid-template-rows: repeat (3, 1fr);
+   grid-column-gap: 0px;
+   grid-row-gap: 0px;
+   padding: 20px; 
+  }
+
+  input[type=text], [type=email], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid rgb(73, 167, 120);
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+
+  .form-button {
+    background: none;
+    border: 1px solid rgb(73, 167, 120);
+    color: rgb(73, 167, 120);
+    border-radius: 20px;
+    font-size: 15px;
+    padding: 5px;
+    text-decoration: none;
+    box-sizing: border-box;
+    cursor: pointer;
+    width: 70px;
+  }
+
+  .form-button:hover {
+    background:rgb(73, 167, 120);
+    color: #ffffff;
+    font-weight: bold;
+  }
+
+  .signup { 
+    grid-area: 1 / 1 / 4 / 5;
+  }
+
+  .signup fieldset {
+    padding: 30px 60px;
+  }
+
+  legend {
+    color: rgb(73, 167, 120);
+  }
+
+  .accordion { 
+    grid-area: 4 / 1 / 4 / 5;
+  }
+
+  .accordion__button {
+    display: block;
+    width: 100%;
+    padding: 15px;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    background: #555454;
+    color: #ffffff;
+    text-align: left;
+    transition: background 0.2s;
+    font-size: 17px;
+  }
+
+  .accordion__button::after {
+    content: '\002B';
+    float: right; 
+    transform: scale(1.5);
+  }
+
+  .accordion__button--active { 
+    background: #555555;
+  }
+
+  .accordion__button--active::after { 
+    content: '\2212';
+  }
+
+  .accordion__content {
+    overflow: hidden;
+    max-height: 0;
+    transition: max-height 0.5s;
+    padding: 0 15px;
+    background: #eeeeee;
+    font-size: 14px;
+  }
+
+  .accordion__content p {
+    padding: 20px;
+  }
+
+  footer {
+    width: 100%; 
+  }
+
+  .footer-signature {
+    padding: 30px;
+    text-align: center;
+    color:rgb(73, 167, 120);
+  }
+
+/*Tablet*/
+@media (min-width: 668px) and (max-width: 1024px) {
+  .text {
+    position: absolute;
+    color: #ffffff;
+    line-height: 1.5;
+    text-align: center;
+  }
+  
+  .menu-button {
+    background: none;
+    border: solid #ffffff;
+    color: #ffffff;
+    border-radius: 40px;
+    font-size: 15px;
+    padding: 20px;
+    text-decoration: none;
+    box-sizing: border-box;
+    cursor: pointer;
+  }
+  
+  .menu-button:hover {
+    background: #ffffff; 
+    color: #333333;
+    font-weight: bold;
+  }
+  
+  .text h1 {
+    font-size: 80px;
+  }
+  
+  .text p {
+    font-size: 18px;
+    padding: 20px;
+    margin-bottom: 40px; 
+  }
+}
+
+/*Destop*/
+@media (min-width: 1025px) {
+  .text {
+    position: absolute;
+    color: #ffffff;
+    line-height: 1.5;
+    text-align: center;
+  }
+  
+  .menu-button {
```

Comment:
This is not neccessary to have here right? It is the same for the mediaquerie as the regular css `class .menu-button{}`. 

This was something that I found hard. Not to repeat myself.